src/custom-providers/azext_custom_providers/_help.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['custom-providers resource-provider'] = """
+    type: group
+    short-summary: Commands to manage custom resource provider.
+"""
```

Comment:
Redundant indentations. We can use <kbd>Ctrl</kbd>+<kbd>A</kbd> to select all, then press <kbd>Shift</kbd>+<kbd>Tab</kbd> to remove the indentations. (https://github.com/Azure/azure-cli-extensions/pull/1545#discussion_r417089561)